src/pages/landing/ebooks/components/_get-ebook.tsx

```diff
@@ -147,17 +147,17 @@ const Line = styled.div`
     background-color: ${(props) => (props.color ? props.color : 'var(--color-grey-6)')};
 `
 
-const StyledText = styled(Text)`
+const StyledText = styled(Text)<{ tabletFontSize?: string }>`
     color: ${(props) => (props.color ? props.color : 'var(--color-grey-6)')};
 
-    @media ${(props) => device.tabletL && props.notedBox} {
+    @media ${device.tabletL} {
```

Comment:
dont we need this `notedBox` prop?

Reply:
so in that case this query never sets on styles, isn't it?

Reply:
no we don't use this